test-data/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -0,0 +1,236 @@
+-   case: from_queryset_with_base_manager
```

Comment:
According to the test, there has to be some kind of magic somewhere which adds `MyModel` as a generic argument for `NewManager`. Could we get rid of that? 
So, what I'm trying to say is that tests are a bit wrong. `from_queryset` line should be
```
class ModelQuerySet(models.QuerySet[MyModel]):
   ...
NewManager = BaseManager[MyModel].from_queryset(ModelQuerySet)
```
and current line should error out with `Need type annotation for` like it's done here https://github.com/typeddjango/django-stubs/blob/newsemanal-progress/test-data/typecheck/managers/test_managers.yml#L128
and `NewManager` in the current setup should give `Any` as a generic argument, see https://github.com/typeddjango/django-stubs/blob/newsemanal-progress/test-data/typecheck/managers/test_managers.yml#L106

Could be done in a separate PR though.